src/Validation/Validator.php

```diff
@@ -211,6 +219,43 @@ public function getProvider($name)
     }
 
     /**
+     * Returns the default provider stored under that name if it exists.
+     *
+     * @param string $name The name under which the provider should be retrieved.
+     * @return object|string|null
```

Comment:
I copied the docblock from https://github.com/lilHermit/cakephp/blob/2b580b05c0ee869daf5b38709d37756c5e01f90a/src/Validation/Validator.php#L270 ;-)